### ISSUE TITLE
serology doesn't recognize broad/split relationships

### ISSUE THREAD

**Issue by mmaiers-nmdp** [nmdp-bioinformatics/py-ard#77]
In addition to the relationships between individual alleles and serologic types (rel_dna_ser) py-ard also needs to take into account the broad split relationships.

For instance, DR6 is a broad specificity with splits of DR13 and DR14.
Yet looking up DR6 returns only a few alleles that somehow map only to the broad.

```
>>> ard.redux_gl('DR6', 'lgx')
'DRB1*14:16/DRB1*14:17/DRB1*14:18/DRB1*14:186'
```

The desired behavior is for DR6 to return these alleles **plus** all of the alleles that are in the expansion of DR13 and DR14 as well.

The broad-split relationships are defined here:

https://raw.githubusercontent.com/ANHIG/IMGTHLA/Latest/wmda/rel_ser_ser.txt

**Comment by pbashyal-nmdp** [comment]
There's a mapping in https://github.com/nmdp-bioinformatics/py-ard/blob/master/pyard/broad_splits.py but looks like it has the older format for serology with `*`.

**Comment by pbashyal-nmdp** [comment]
Closed by #78
